.projenrc.ts

```diff
@@ -1,12 +1,193 @@
\ No newline at end of file
+
+new SourceUpdate(project, {
+  name: 'config-xsd',
+  source: {
+    url: 'https://docs.aws.amazon.com/amazon-mq/latest/developer-guide/amazon-mq-broker-configuration-parameters.html',
+    xsdUrlPrefix: 'https://docs.aws.amazon.com/amazon-mq/latest/developer-guide/samples/'
+  },
+  cwd: 'sources'
+});
+
+project.gitignore.exclude("__MACOSX/");
+
+const xsd2jsii = new TypeScriptWorkspace({
+  parent: project,
+  private: true,
+  stability,
+  name: "@cdklabs/xsd2jsii",
+  packageName: "xsd2jsii",
+  bin: {
+    xsd2jsii: "bin/xsd2jsii",
+  },
+  deps: ["json5", "xml2js", "@faker-js/faker"],
+  bundledDeps: ["json5", "xml2js", "@faker-js/faker"],
+  devDeps: ["@types/xml2js", 'xsd-schema-validator'],
+  prettier: true,
+  tsconfigDev: {
+    include: ["bin/**/*.ts"],
+  },
+});
+
+xsd2jsii.addFields({ stability });
+
+const schemas = fs.readdirSync("./sources");
+
+const schemaReadmeTemplate = fs.readFileSync(
+  "./templates/README_cdk-amazonmq-activemq-config.TEMPLATE.md",
+  "utf-8"
+);
+
+const generatedTestTsTemplate = fs.readFileSync(
+  "./templates/generated.test.ts.template",
+  "utf-8"
+);
+
+const versionRegex = /^amazon-mq-active-mq-(\d+)\.(\d+)\.(\d+)\.xsd$/;
+
+for (const schema of schemas) {
+  const match = versionRegex.exec(schema);
+  if (!match) {
+    continue;
+  }
+
+  const [_, MAJOR, MINOR, PATCH] = match;
+
+  const schemaProject = new TypeScriptWorkspace({
+    parent: project,
+    stability,
+    name: `@cdklabs/cdk-amazonmq-activemq-config-v${MAJOR}-${MINOR}-${PATCH}`,
+    authorName: "AWS",
+    authorEmail: "cdk-amazonmq-maintainers@amazon.com",
+    repository:
+      "https://github.com/cdklabs/cdk-amazonmq-activemq-configs",
+    devDeps: [xsd2jsii.package.packageName, 'xsd-schema-validator'],
+    prettier: true,
+    jest: true,
+    disableTsconfig: true,
+    releasableCommits: ReleasableCommits.featuresAndFixes("."),
+  });
+
+  new TextFile(schemaProject, "README.md", {
+    marker: true,
+    committed: true,
+    lines: schemaReadmeTemplate.replaceAll('{{VERSION}}', `${MAJOR}.${MINOR}.${PATCH}`).split('\n')
+  });
+
+  new Jsii(schemaProject, {
+    stability,
+    jsiiVersion: "~5.7.0",
+    publishToPypi: {
+      distName: `cdklabs.cdk-amazonmq-activemq-config-v${MAJOR}-${MINOR}-${PATCH}`,
+      module: `cdklabs.cdk-amazonmq-activemq-config-v${MAJOR}-${MINOR}-${PATCH}`,
+    },
+    publishToNuget: {
+      dotNetNamespace: `Cdklabs.CdkAmazonmq.ActiveMqConfig.V${MAJOR}_${MINOR}_${PATCH}`,
+      packageId: `Cdklabs.CdkAmazonmq.ActiveMqConfig.V${MAJOR}-${MINOR}-${PATCH}`,
+    },
+    publishToMaven: {
+      javaPackage: `io.github.cdklabs.cdkamazonmqactivemqconfig.v${MAJOR}_${MINOR}_${PATCH}`,
+      mavenGroupId: "io.github.cdklabs",
+      mavenArtifactId: `cdk-amazonmq-activemq-config-v${MAJOR}-${MINOR}-${PATCH}`,
+    },
+    // publishToGo: {
+    //   moduleName: `github.com/cdklabs/cdk-amazonmq-activemq-config-v${MAJOR}_${MINOR}_${PATCH}`,
+    // },
```

Comment:
Is this expected to comment out?

Reply:
For the time being - yes. We still need to identify the approach to distributing the Go packages. We can put it in a single Go repo (say, `github.com/cdklabs/cdk-amazonmq-activemq-config-go`) and there set up a structure. We wouldn't, however, want to autogenerate GitHub repositories per Spring XML schemas. 

We intentionally (for now) will not be supporting Go.